lighthouse-core/gather/gatherers/http-redirect.js

```diff
@@ -36,39 +36,20 @@ class HTTPRedirect extends Gatherer {
   }
 
   afterPass(options) {
-    // Reset the options.
-    options.url = this._preRedirectURL;
-
-    // Allow override for faster testing.
-    const timeout = options._testTimeout || 10000;
-
-    const securityPromise = options.driver.getSecurityState()
-      .then(state => {
-        return {
-          value: state.schemeIsCryptographic
-        };
+    let isRedirect = false;
+
+    if(options.driver._httpsArr.length !== 0) {
```

Comment:
Smoketests are failing if it's not checked for, as it results to `undefined`. And yeah, he also is I think. Is there something I need to change here?